packages/suite/src/utils/wallet/transactionUtils.ts

```diff
@@ -301,9 +301,9 @@ export const isTxUnknown = (transaction: WalletAccountTransaction) => {
     );
 };
 
-export const getFeeRate = (tx: AccountTransaction, convertToDecimals?: number) => {
+export const getFeeRate = (tx: AccountTransaction, decimals?: number) => {
     // calculate fee rate, TODO: add this to blockchain-link tx details
-    const fee = convertToDecimals ? amountToSatoshi(tx.fee, convertToDecimals) : tx.fee;
+    const fee = typeof decimals === 'number' ? amountToSatoshi(tx.fee, decimals) : tx.fee;
```

Comment:
just quick question, shouldn't it be handled by typescript, when you pass something other than a number then type-check will fail right? Or is there added advantage in doing manual check?